### PR TITLE
gap on the right is fixed and the text and icon design is following t…

### PR DESCRIPTION
**Pull request by Amutha37**
…he latest request

## Overview of the Pull Request:
- Text for the `Distance travelled by mode of transport`  changed according to latest figma design. 
- Spacing issue is fixed

## link to Trello card or Github issue
Heading text and icon design changes trello : 

https://trello.com/c/TMOXpyaD/345-distance-travelled-by-mode-of-transportpng

Trello card for spacing issue :

https://trello.com/c/Deh49NQL/330-inconsistent-spacing

## How Has This Been Tested?
Run locally and it fits desktop and mobile.

## Pull Request Checklist:

Make sure the following checkboxes`[ ]` are checked with `x` before sending your PR -- thank you!

- [ x] Have you included a link Trello card / Github issue and written sufficient description to help others review your work?
- [ x] Is your pull request up-to-date with `main` branch?
- [ x] Have you checked That code is well formatted? Did `npx prettier --check .` return no warnings/errors?
- [x ] Have you written instructions on how to test that your changes work as expected?
- [ x] Have you tested your work thoroughly on your local machine to ensure you are not breaking anything?

### CODE REVIEW

File: components/ResultsPageComponents/DistanceTravelledMode/DistanceTravelledMode.jsx [hunk 1]
```diff
@@ -12,124 +12,136 @@ export default function DistanceTravelledMode({ data }) {
...
-              <Text fontWeight={600} fontSize="23px">
-                Individual Method
-              </Text>
-            </Flex>
+          <Flex direction="column">
+            <Text fontWeight={600} color="#022640" fontSize="22px">
+              Active/Public/Shared Methods:
+            </Text>
+            <Text color="#03385F">
+              Lower carbon emission per kilometre travel per person
```
Comment (hqtan) [inline components/ResultsPageComponents/DistanceTravelledMode/DistanceTravelledMode.jsx:49]: smaller grammar fix:
```suggestion
              Lower carbon emission per kilometres travelled per person
```

**Review by hqtan — CHANGES_REQUESTED** [review]
Awesome work @Amutha37 ! 🙌🏽 

Apart from one small grammar fix, I have no other concerns with this PR 👍🏽

**Review by OlaShabalina — COMMENTED** [review]
Well done @Amutha37! If @hqtan is happy with a separate PR for grammar, we can merge.

**Review by hqtan — APPROVED** [review]

**Review by hqtan — APPROVED** [review]
nice work @Amutha37 ! 👍🏽

**Pull request merged**